### ISSUE TITLE
[Flogo Web] Building serverless App doesn't result in executable file

### ISSUE THREAD

**Issue by retgits** [TIBCOSoftware/flogo#361]
**Current behavior (how does the issue manifest):** With the introduction of the option to build a serverless app from the Web UI, I want to upload the zip I get after building directly to AWS Lambda. When I do that, executing the function results in an error `fork/exec /var/task/handler: no such file or directory: PathError` (the handler generated by the Web UI is correctly called handler)

**Expected behavior:** Running the function should work properly

**Minimal steps to reproduce the problem (not required if feature enhancement):**

* Create an app with the Lambda trigger
* Build the app from the Web UI
* Upload the lambda_trigger.zip from the previous step to AWS Lambda

**Please tell us about your environment (Operating system, docker version, browser & web ui version, etc):** Flogo Web 0.5.5

**Flogo version (CLI & contrib/lib. If unknown, leave empty or state unknown):** 0.5.5

**Additional information you deem important (e.g. issue happens only occasionally):** Using the build steps from the CLI (after exporting the app.json), I've been able to generate a working app

**Comment by mellistibco** [comment]
The issue seems to be related to the fact that the bin build in the docker image (alpine base) is producing a dynamically linked bin:

```
handler: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib/ld-musl-x86_64.so.1, with debug_info, not stripped
```

A similar issue came up here: https://github.com/serverless/serverless/issues/4710

**Comment by fcastill** [comment]
Also related to #313

**Comment by mellistibco** [comment]
@retgits this should be resolved in the latest web ui build (0.5.6). can you confirm?

**Comment by retgits** [comment]
Hi, my name is @retgits and I confirm this is solved 🤣
